bikeshed/__init__.py

```diff
@@ -1248,6 +1262,7 @@ def preprocess(self):
         processHeadings(self, "all") # again
         addTOCSection(self)
         addSelfLinks(self)
+        processBiblioLinks(self) # again, as biblio links could be added by addIndexSection().
```

Comment:
That's not too bad, actually. I'll throw another patch at you shortly; unfortunately, it involves rebasing a lot of tests. :/